src/connector/src/schema/schema_registry/client.rs

```diff
@@ -135,12 +138,25 @@ impl Client {
             client: self.inner.clone(),
             path: path.iter().map(|p| p.to_string()).collect_vec(),
         });
+
+        let retry_strategy = ExponentialBackoff::from_millis(100)
```

Comment:
How about making the backoff duration, factor and attempts configurable via a developer config so that we can tune the config without rebuilding an image just in case?